Index: app/src/main/java/minhfx03283/funix/arrayadaptertest/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/minhfx03283/funix/arrayadaptertest/MainActivity.java	(revision 41af8f7477230d1e0550a240cfe96aa264e77b89)
+++ app/src/main/java/minhfx03283/funix/arrayadaptertest/MainActivity.java	(date 1614248170000)
@@ -60,7 +60,7 @@
         // 2. What process involves treating rubber with sulfur to harden it? (Correct Answer is "Vulcanizing")
         QuizType2 q2 = new QuizType2();
         q2.setQuiz("2. " + getResources().getString(R.string.q2));
-        Set<String> q2Answer = new HashSet<String>(Arrays.asList(getResources().getString(R.string.q2_ans).toLowerCase()));
+        Set<String> q2Answer = new HashSet<String>(Arrays.asList(getResources().getString(R.string.q2_ans)));
         q2.setCorrectAnswer(q2Answer);
 
         // 3. Name two different organelles of a eukaryotic cell. (Correct Answers are #1 (Ribosomes) and #3 (Golgi Apparatus))
@@ -85,7 +85,7 @@
         // (Correct Answer is "Gravity")
         QuizType2 q4 = new QuizType2();
         q4.setQuiz(getResources().getString(R.string.q4));
-        Set<String> q4Answer = new HashSet<>(Arrays.asList(getResources().getString(R.string.q4_ans).toLowerCase()));
+        Set<String> q4Answer = new HashSet<>(Arrays.asList(getResources().getString(R.string.q4_ans)));
         q4.setCorrectAnswer(q4Answer);
 
 //        5 What type of trees yield the resin used to produce turpentine?
@@ -107,8 +107,8 @@
         QuizType2 q6 = new QuizType2();
         q6.setQuiz("6. " + getResources().getString(R.string.q6));
         Set<String> q6Answer = new HashSet<>(Arrays.asList(
-                getResources().getString(R.string.q6_ans1).toLowerCase(),
-                getResources().getString(R.string.q6_ans2).toLowerCase()));
+                getResources().getString(R.string.q6_ans1),
+                getResources().getString(R.string.q6_ans2)));
         q6.setCorrectAnswer(q6Answer);
 
         // 7. Which two planets have one or more moons? (Correct Answers are #3 (Earth) and #4 (Pluto))
@@ -131,9 +131,9 @@
 
         //8. Where in the human body would you find the scaphoid bone? (Correct Answer is "Wrist")
         QuizType2 q8 = new QuizType2();
-        q8.setQuiz("8. " + getResources().getString(R.string.q8).toLowerCase());
+        q8.setQuiz("8. " + getResources().getString(R.string.q8));
         q8.setCorrectAnswer(new HashSet<String>(
-                Arrays.asList(getResources().getString(R.string.q8_ans).toLowerCase())));
+                Arrays.asList(getResources().getString(R.string.q8_ans))));
 
         //9. Which grow upwards Stalactites or Stalagmites? (Correct Answers is #2 "Stalagmites")
         //Stalactites
@@ -147,7 +147,7 @@
         //10. What process involves heating an ore to obtain a metal? (Correct Answer is "Smelting")
         QuizType2 q10 = new QuizType2();
         q10.setQuiz("10. " + getResources().getString(R.string.q10));
-        q10.setCorrectAnswer(new HashSet<String>(Arrays.asList(getResources().getString(R.string.q10_ans).toLowerCase())));
+        q10.setCorrectAnswer(new HashSet<String>(Arrays.asList(getResources().getString(R.string.q10_ans))));
 
         quizzes.add(q1);
         quizzes.add(q2);
Index: app/src/main/res/values/strings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/strings.xml	(revision 41af8f7477230d1e0550a240cfe96aa264e77b89)
+++ app/src/main/res/values/strings.xml	(date 1614248170000)
@@ -41,6 +41,6 @@
     <string name="q10_ans">Smelting</string>
     <string name="perfect">Perfect!</string>
     <string name="try_again">Try again!</string>
-    <string name="scored_1">You score</string>
+    <string name="scored_1">You scored</string>
     <string name="scored_2">out of</string>
 </resources>
\ No newline at end of file
